HPCC-18964 Add translations to Login.html

miguelvazq opened the pull request:
Signed-off-by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [ ] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

HPCCSmoketest commented:
This PR (11791) is in the 4/5 position of the queue. Estimated start time is after ~2.0 hour(s)

HPCCSmoketest commented:
This PR (11791) is in the 3/4 position of the queue. Estimated start time is after ~1.5 hour(s)

HPCCSmoketest commented:
This PR (11791) is in the 2/3 position of the queue. Estimated start time is after ~1.0 hour(s)

HPCCSmoketest commented:
5/6. Process of PR-11791, label: HPCC-18964 starts now. Estimated completion time is ~ 0.5 hour(s)

HPCCSmoketest commented:
Automated Smoketest: :x:
OS: centos 7.4.1708 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 466a5c6b679614bb4a878fb3a57f7d93a6bf6617
Build: success
ECL Watch:  Rebuilding Site
Rebuild: success
Install hpccsystems-platform-community_7.0.0-rc2.el7.x86_64.rpm
HPCC Start: Fail
HPCC start:
Starting mydafilesrv ...     [   OK    ]  
Starting mydali ...          [   OK    ]  
Starting mydfuserver ...     [   OK    ]  
Starting myeclagent ...      [   OK    ]  
Starting myeclccserver ...   [   OK    ]  
Starting myeclscheduler ...  [   OK    ]  
Starting myesp ...           [   OK    ]  
Starting myroxie ...         [   OK    ]  
Starting mysasha ...         [   OK    ]  
Starting mythor ...          [   OK    ]  
 
HPCC status:
mydafilesrv     ( pid    23985 ) is running ...
mydali          ( pid    24310 ) is running ...
mydfuserver     ( pid    24683 ) is running ...
myeclagent      ( pid    25031 ) is running ...
myeclccserver   ( pid    25366 ) is running ...
myeclscheduler  ( pid    25714 ) is running ...
myesp           ( pid    26052 ) is running ...
myroxie         ( pid    26462 ) is running ...
mysasha         ( pid    26870 ) is running ...
mythor          ( pid    27472 ) is running with 4 slave process(es) ...
HPCC Uninstall: OK
Time stats:

| Prep time | Build time | Package time | Install time | Start time | Test time | Stop time | Summary | 
|---|---|---|---|---|---|---|---|
|   26 sec (00:00:26) |  188 sec (00:03:08) |  169 sec (00:02:49) |   22 sec (00:00:22) |   45 sec (00:00:45) |    0 sec (00:00:00) |    0 sec (00:00:00) |  450 sec (00:07:30) |

HPCCSmoketest commented:
This PR (11791) is in the 3/3 position of the queue. Estimated start time is after ~1.5 hour(s)

HPCCSmoketest commented:
This PR (11791) is in the 2/2 position of the queue. Estimated start time is after ~1.0 hour(s)

HPCCSmoketest commented:
4/4. Process of PR-11791, label: HPCC-18964 starts now. Estimated completion time is ~ 0.5 hour(s)

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
OS: centos 7.4.1708 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 466a5c6b679614bb4a878fb3a57f7d93a6bf6617
Build: success
ECL Watch:  Rebuilding Site
Rebuild: success
Install hpccsystems-platform-community_7.0.0-rc2.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |95 |95 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |806 |806 |0 |
|test (thor) |730 |730 |0 |
|test (roxie) |879 |879 |0 |


HPCC Stop: OK
HPCC Uninstall: OK
Time stats:

| Prep time | Build time | Package time | Install time | Start time | Test time | Stop time | Summary | 
|---|---|---|---|---|---|---|---|
|   23 sec (00:00:23) |  189 sec (00:03:09) |  165 sec (00:02:45) |   20 sec (00:00:20) |   42 sec (00:00:42) | 1513 sec (00:25:13) |   20 sec (00:00:20) | 1972 sec (00:32:52) |

richardkchapman commented on esp/src/lws.config.js:
```diff
@@ -1,4 +1,4 @@
-const debugServerIP = "192.168.99.103";
+const debugServerIP = "10.176.152.166";
```
Doesn't deel like information that should be checked in (even if it's an internal IP)

miguelvazq replied:
Agreed will remove and fix up the commit message as well.

Commit:
HPCC-18964 Add translations to Login.html

Add ability to embed translations into session based pages.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

miguelvazq commented:
@GordonSmith Please review. @kunalaswani FYI seems there was some translations that missed the cut.

HPCCSmoketest commented:
This PR (11791) is in the 1/2 position of the queue. Estimated start time is after ~0.5 hour(s)

HPCCSmoketest commented:
2/3. Process of PR-11791, label: HPCC-18964 starts now. Estimated completion time is ~ 0.5 hour(s)

GordonSmith reviewed (approved):
Looks fine.

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
OS: centos 7.4.1708 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 7e3628ce9623bf63766dc036124f142069a27b73
Build: success
ECL Watch:  Rebuilding Site
Rebuild: success
Install hpccsystems-platform-community_7.0.0-rc2.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |95 |95 |0 |0 |0 |
|wutoolTest(Dali) |19 |19 |0 |0 |0 |
|wutoolTest(Cassandra) |19 |19 |0 |0 |0 |



Regression test result:

| phase | total | pass | fail | 
|---|---|---|---|
|setup (hthor) |11 |11 |0 |
|setup (thor) |11 |11 |0 |
|setup (roxie) |11 |11 |0 |
|test (hthor) |806 |806 |0 |
|test (thor) |730 |730 |0 |
|test (roxie) |879 |879 |0 |


HPCC Stop: OK
HPCC Uninstall: OK
Time stats:

| Prep time | Build time | Package time | Install time | Start time | Test time | Stop time | Summary | 
|---|---|---|---|---|---|---|---|
|   43 sec (00:00:43) |  185 sec (00:03:05) |  168 sec (00:02:48) |   20 sec (00:00:20) |   42 sec (00:00:42) | 1481 sec (00:24:41) |   19 sec (00:00:19) | 1958 sec (00:32:38) |

miguelvazq commented:
@richardkchapman ready to merge.

richardkchapman commented:
Do we need to add anything to the recently-done translation files?

GordonSmith commented:
@kunalaswani - they will need another iteration for new words (and will default to English in the meantime)

kunalaswani commented:
@richardkchapman once all of them are in i will need to do one more run for words that have been added last min yes. I can open a child ticket if you think that is the best way to go about it.
